app/mainwindow2.h

```diff
@@ -91,6 +91,8 @@ public slots:
 
     void importMovie();
 
+    void lockWidgets( bool shouldLock );
```

Comment:
I see. This makes me wonder: Does placing a method in the slots section actually do anything or is it just semantic sugar for stuff like code completion? I mean, if it actually did something we probably should have gotten warnings or bugs by now… Anyways, I agree that we should fix all of those sooner or later either way, even if it only makes the code more comprehensible.